src/pipelines/stages/expression/ExtractInlineSolveState.ts

```diff
@@ -3,23 +3,28 @@ import { IExpressionProcessorState } from "@/pipelines/stages/expression/state/I
 
 export class ExtractInlineSolveStage extends BaseStatefulPipelineStage<
 	IExpressionProcessorState,
-	string
+	string[]
 > {
-	// Matches for either s`EXPRESSION` and will return the first instance.
-	private inlineSolveRegex = new RegExp(/s`([^`]*)`/);
+	// Matches for either s`EXPRESSION`.
+	private inlineSolveRegex = new RegExp(/s`([^`]*)`/g);
 
 	protected execute(
 		state: IExpressionProcessorState,
-		request: string
-	): string {
+		request: string[]
+	): string[] {
 		// IMPORTANT: We match the raw input expression and not the processed expression because
 		// 	          in cases where their is markdown elments e.g.lists, qouotes, etc...
```

Comment:
Whoops, this is my bad 😅 
```suggestion
		// 	          in cases where their is markdown elments e.g. lists, quotes, etc...
```

Reply:
I can submit another PR after this to clean up any spelling errors. Just don't want to clutter up this one.